ThunkLibs/libX11/libX11_interface.cpp

```diff
@@ -0,0 +1,526 @@
+#include <common/GeneratorInterface.h>
+
+#include <X11/Xlib.h>
+#include <X11/Xutil.h>
+#include <X11/Xresource.h>
+
+#include <X11/Xlibint.h>
+#include <X11/XKBlib.h>
+
+template<auto>
+struct fex_gen_config {
+    unsigned version = 1;
```

Comment:
I believe this should actually be 6. Typo in the current version of the python scripts.

Reply:
Fixed as a dedicated commit before migrating.